pkg/blueprint/customizations.go

```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
 	"reflect"
 	"strings"
+
+	"golang.org/x/exp/slices"
```

Comment:
Part of Go 1.21, at least the used `Reverse` [function](https://pkg.go.dev/slices#Reverse).

Reply:
As the title mentions, this is a year-old branch :laughing: 
But thanks, I'll try to get back to this soon and rebase.  It's going to need a lot of work.